### DISCUSSION TITLE
安装软件失败

### DISCUSSION THREAD

**Discussion by MatterMu** [secretflow/secretflow#1207] [Q&A]
Collecting secretflow
  Using cached secretflow-1.5.0.dev20240304-cp311-cp311-manylinux2014_x86_64.whl.metadata (4.7 kB)
  Using cached secretflow-0.7.7b0.tar.gz (163 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [10 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-_cimlk27/secretflow_09492ffcbb2b47aebb40f920d2954c11/setup.py", line 100, in <module>
          install_requires=read_requirements(),
                           ^^^^^^^^^^^^^^^^^^^
        File "/tmp/pip-install-_cimlk27/secretflow_09492ffcbb2b47aebb40f920d2954c11/setup.py", line 17, in read_requirements
          with open('./requirements.txt') as file:
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
      FileNotFoundError: [Errno 2] No such file or directory: './requirements.txt'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
求教怎么解决

**Comment by Chrisdehe** [comment]
请问你的python版本是多少呢？
请按照1.5.0正式版[安装部署](https://www.secretflow.org.cn/zh-CN/docs/secretflow/v1.5.0b0/getting_started/installation)的版本配置来操作
